src/js/utils/window_watchers.js

```diff
@@ -37,23 +37,25 @@ windowWatcherUtil.bindWatchers = function(windowObject, watchersObject) {
 
 // 'Windows' here could be array or single window reference
 windowWatcherUtil.passValue = function(windows, key, val) {
-    let applyBinding = function(win, key, val) {
+    this.applyBinding(windows, key, val);
+}
+
+windowWatcherUtil.applyBinding = function(win, key, val) {
```

Comment:
Keeping this as a local scoped function seems to be an easier to understand way of doing this.

Reply:
Fixed as per review